crate/server/src/core/cover_crypt/update_policy.rs

```diff
@@ -265,3 +220,70 @@ async fn refresh_all_user_decryption_keys(
 
     Ok(())
 }
+
+/// Update a Covercrypt policy based on the specified action and master private key.
+///
+/// This function takes an `EditPolicyAction` and a master private key as input and performs the
+/// corresponding policy updates. It retrieves the current policy from the master private key's
+/// attributes and applies the requested action, resulting in a new policy and a list of attributes
+/// that were updated or modified.
+///
+/// # Parameters
+///
+/// - `action`: An `EditPolicyAction` enum representing the action to perform on the policy.
+/// - `master_private_key`: A reference to the master private key object from which the policy
+///   attributes are retrieved.
```

Comment:
Avoid redundancy with function signature
```suggestion
/// - `action`: action to perform on the policy.
/// - `master_private_key`: master private key for which the policy is updated.
```